python_libs/requirements

```diff
@@ -1,25 +1,23 @@
-absl-py==0.2.2
-astor==0.6.2
-backports.weakref==1.0.post1
-bleach==1.5.0
-decorator==4.3.0
-enum34==1.1.6
-funcsigs==1.0.2
-futures==3.2.0
-gast==0.2.0
-grpcio==1.12.1
-html5lib==0.9999999
-Markdown==2.6.11
-mock==2.0.0
-networkx==1.11
-numpy==1.14.5
-pbr==4.0.4
-protobuf==3.6.0
-scikit-learn==0.19.1
-scipy==1.1.0
-six==1.11.0
-sklearn==0.0
-tensorboard==1.8.0
-tensorflow==1.8.0
-termcolor==1.1.0
-Werkzeug==0.14.1
+absl-py-1.4.0
+astor-0.8.1
+backports.weakref-1.0.post1
+bleach-6.0.0
+decorator-5.1.1
+enum34-1.1.10
+funcsigs-1.0.2
+gast-0.4.0
+grpcio-1.56.0
+html5lib-1.1
+Markdown-3.4.3
+mock-5.1.0
+networkx-3.1
+numpy-1.24.3
+pbr-5.11.1
+protobuf-4.23.4
+scikit-learn-1.3.0
+scipy-1.11.1
+six-1.16.0
+tensorboard-2.13.0
+tensorflow-2.13.0
```

Comment:
Shall we test the train command before updating the tensorflow version?

Reply:
Done.